packages/cells/src/widget.ts

```diff
@@ -492,6 +495,30 @@ export class Cell<T extends ICellModel = ICellModel> extends Widget {
     this.inputHidden = !!jupyter.source_hidden;
   }
 
+  /**
+   * Load the tag state from the model.
+   */
+  loadTagsState(): void {
```

Comment:
Can we make this method private, or is your intent to call it in extensions?


```suggestion
  private _loadTagsState(): void {
```

You could still use `widget['_loadTagsState']()` in tests then.